e2e/tests/functional/plugins/plot/stackedPlot.e2e.spec.js

```diff
@@ -76,4 +83,20 @@ test.describe('Stacked Plot', () => {
         await expect(page.locator('.js-elements-pool__tree >> text=swg b')).toHaveCount(0);
         await expect(page.locator('.js-elements-pool__tree >> text=swg c')).toHaveCount(1);
     });
+
+    test('Selecting a child plot shows it\'s properties in the inspector.', async ({ page }) => {
+
+        await page.goto(stackedPlot.url);
+
+        //Click on the 2nd plot canvas - There will be 2 canvases for each swg, so we click on the 3rd canvas
+        await page.locator('canvas').nth(3).click();
```

Comment:
We should improve this selector so we can avoid using `nth(i)`

Reply:
I could try to grab the plot div first, but we will always need to click on the 1st canvas, so it will still be nth(1). But then the 1 is fixed.

Reply:
Can we add a11y to the canvases maybe? aria-label?